### ISSUE TITLE
"mpc" is ambigous.

### ISSUE THREAD

**Issue by chrisalbertson** [rdragos/awesome-mpc#66]
MPC can also stand for "Model Predictive Control".  One has to read the readme past the point of needing to scroll before one sees which use of "MPC" is addressed by this repository.     

Solution:  Place the words "multi party computing" near the top of the page.

Seriously, pretend you think MPC is "model predicitve control" or even "macro particle creation" and see if the readme still makes perfect sense.  It does.  The text is too generic

**Comment by chrisalbertson** [comment]
The problem is not that a member of your community would not know what “MPC” stands for.  But rather, there are at least three unrelated uses of MPC and Google finds your repository for the other uses

It is possible that a person uses the term “MPC” many times daily in his normal work, but it has nothing to do with Multi-Party Computing.

The problem is Google directs to your page for any use of “MPC” and there are two other common uses (1) the Akai MPC musical instrument, (2) Model Predictive control and (3)  your use of it.   This might confuse a lot of musicians who are looking for Open Source “MPC Software" who need to read the entire README.MD before finding your GitHub has nothing to do with this: https://www.akaipro.com/products/mpc-series.html



Same for control engineers,   The page belo.w is all about “MPC” but it would only take you one second to see the page is not for you.
https://www.do-mpc.com/en/latest/theory_mpc.html


I think it is common practice to fully spell out a term before the first use of an acronym.  Most style guise suggest this.

**Comment by rdragos** [comment]
Fair point, you should find this PR now which adds a new sentence in the beginning about MPC
